docs/specifications/tests/Nameserver-TP/nameserver11.md

```diff
@@ -3,91 +3,142 @@
-processed as though the OPTION-CODE was not even there.
+[RFC 6891][RFC 6891, section 6.1.2], section 6.1.2, states that any OPTION-CODE values
+not understood by a responder or requestor MUST be ignored. Unknown OPTION-CODE values
+must be processed as though the OPTION-CODE was not even there.
 
-In this test case, we will query  with an unknown EDNS OPTION-CODE and expect
+In this test case, we will query with an unknown EDNS OPTION-CODE and expect
 that the OPTION-CODE is not present in the response for the query.
 
 ## Scope
 
-It is assumed that *Child Zone* has been tested by [Basic04]. This test
-case will set DEBUG level on messages for non-responsive name servers.
+It is assumed that *Child Zone* has been tested and reported by [Basic04]. This
+test case will just ignore non-responsive name servers or name servers not
+giving a correct DNS response for an authoritative name server.
+
+It is assumed that *Child Zone* has been tested and reported by [Nameserver02].
+Running this test case without running [Nameserver02] can give an incomplete
+report status of *Child Zone*.
 
 ## Inputs
 
 "Child Zone" - The domain name to be tested.
 
-## Ordered description of steps to be taken to execute the test case
+## Summary
+
+Message                                | Level     | Arguments           | Description of when message tag is outputted
+:--------------------------------------|:----------|---------------------|---------------------------------------------
+NS11_UNKNOWN_OPTION_CODE               | WARNING   | ns_ip_list          | The DNS response contains an unknown EDNS option-code
+
+The value in the Level column is the default severity level of the message. The
+severity level can be changed in the [Zonemaster-Engine profile]. Also see the
+[Severity Level Definitions] document.
+
+The argument names in the Arguments column lists the arguments used in the
+message. The argument names are defined in the [argument list].
+
+
+## Test procedure
+
+In this section and unless otherwise specified below, the term "[DNS Query]"
+follows the specification for DNS queries as specified in
+[DNS Query and Response Defaults]. The handling of the DNS responses on the DNS
+queries follow, unless otherwise specified below, what is specified for
+[DNS Response] in the same specification.
 
-1. Create a SOA query for the *Child Zone* with an OPT record with 
-   EDNS OPTION-CODE set to anything other than it is already assigned as in the
-[IANA-DNSSYSTEM-PARAMETERS] and no other EDNS options or flags.
+1. Create the following empty sets:
+   1. Name server IP address ("Unknown Option Code")
 
-2. Obtain the set of name server IP addresses using [Method4] and [Method5]
+2. Create an SOA query for the *Child Zone* with an OPT record with 
+   EDNS OPTION-CODE set to anything other than it is already assigned as in
+   the [IANA-DNSSYSTEM-PARAMETERS] and no other EDNS options or flags.
```

Comment:
Above it is referred to "DNS Query and Response Defaults" but it is not utilized here.
```
2. Create a [EDNS Query] with query type SOA, *Child Zone* as query name and with
EDNS OPTION-CODE set to anything other than it is already assigned as in
the [IANA-DNSSYSTEM-PARAMETERS] and no other EDNS options or flags ("SOA Query").
```

Reply:
Fixed